pages/ethics-tree.md

```diff
@@ -15,3 +15,9 @@ Adhering to the requirements of NCCRI, ERIM has set up [internal review boards (
 ## When do I need an ethical approval?
 
 [!["A decision tree showing when to apply for an ethical approval"](assets/img/ethics-decision-tree.png)](assets/files/ethics-decision-tree.pdf)
+
+## Ethical dilemmas
+
+How to deal with ethical dilemmas is not a simple task,that is why EUR developed a game called [the dilemma game app](https://www.eur.nl/en/about-eur/policy-and-regulations/integrity/research-integrity/dilemma-game), where you can find examples of real ethical dilemmas that reseachers, students, research support staff, etc. have experienced.
+
+For more dedicated support, please do not hesitate to contact [EUR's confidential counsellors](https://my.eur.nl/en/eur-employee/hr/healthy-safe-working/mental-and-social/undesirable-behaviour/confidential-counsellors?check_logged_in=1)
```

Comment:
Thanks for the change! Reading this now, I'm currently not 100% sure about mentioning confidential counselors here. This page refers to ethics approval, and I'd want to avoid an impression that researchers need to contact confidential counselors for their research projects (that can happen, but the linked page says that counselors are for talking about undesirable behavior). Perhaps, we could delete this line for now, and think about where we can mention confidential counselors? (Perhaps in another GitHub Issue?)
```suggestion
```

Reply:
I think you are right, it is a bit out of scope of the page and it is also too hidden if you are actually looking for confidential counselors, so I agree, for now we can leave the ERIM page as it is for now (that I assume will be updated at some point with Kim's name for example), thank you for the thorough check, appreciated! :)